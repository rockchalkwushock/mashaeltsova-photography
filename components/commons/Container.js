import { styles } from '../../lib'

const Container = ({ children, className }) => (
  <div className={className}>
    {children}
    <style jsx>{`
      div {
        flex: 1 1 auto;
      }
      .copyright {
        font-family: ${styles.fonts.courgette};
      }
      @media (min-width: 768px) {
        .copyright {
          text-align: right;
        }
        .social {
          text-align: left;
        }
      }
    `}</style>
  </div>
)

export default Container
