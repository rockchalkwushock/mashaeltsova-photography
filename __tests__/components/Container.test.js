import React from 'react'
import { render } from 'enzyme'

import Container from '../../components/commons/Container'

describe('Component: Container', () => {
  test('Renders without exploding as .copyright', () => {
    const tree = render(<Container className="copyright" />)
    expect(tree).toMatchSnapshot()
  })
  test('Renders without exploding as .social', () => {
    const tree = render(<Container className="social" />)
    expect(tree).toMatchSnapshot()
  })
})
