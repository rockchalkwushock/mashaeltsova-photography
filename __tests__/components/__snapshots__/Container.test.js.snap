// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Container Renders without exploding as .copyright 1`] = `
<div
  class="jsx-590062731 copyright"
/>
`;

exports[`Component: Container Renders without exploding as .social 1`] = `
<div
  class="jsx-590062731 social"
/>
`;
